sql/core/src/test/scala/org/apache/spark/sql/internal/SQLConfSuite.scala

```diff
@@ -320,4 +320,16 @@ class SQLConfSuite extends QueryTest with SharedSparkSession {
     assert(e2.getMessage.contains("spark.sql.shuffle.partitions"))
   }
 
+  test("set removed configs to non-default values") {
+    Seq(
+      "spark.sql.fromJsonForceNullableSchema" -> false,
```

Comment:
shall we only test one config? Otherwise we may need to update this test again and again when we remove more configs in the future.

In `cannot set/unset static SQL conf` we only test one config too.

Reply:
I agree. I will test only `spark.sql.fromJsonForceNullableSchema`